commands/fetch_templates.go

```diff
@@ -137,3 +186,29 @@ func createPath(relativePath string, perms os.FileMode) error {
 	err := os.MkdirAll(dir, perms)
 	return err
 }
+
+// Tells whether the language can be expanded from the zip or not
+// availableLanguages map keeps track of which languages we know to be okay to copy.
+// overwrite flag will allow to force copy the language template
+func canWriteLanguage(availableLanguages *map[string]bool, language string, overwrite bool) bool {
+	canWrite := false
+	if len(language) > 0 {
+		if _, ok := (*availableLanguages)[language]; ok {
```

Comment:
Please add `if availableLanguages != nil` and change `ok` to `exists` or `found`